platform/darwin/src/MGLFoundation.h

```diff
@@ -3,10 +3,3 @@
 #import <Foundation/Foundation.h>
 
 #define MGL_EXPORT __attribute__((visibility ("default")))
-
-/* Using a compound statement (GNU Extension, supported by clang) */
-#define MGL_OBJC_DYNAMIC_CAST(object, type) \
```

Comment:
This macro was previously in the public API. Moving it to a private header would be a backwards-incompatible change, except that a missing `*` above means that this macro was never documented in the first place.

Reply:
Yeah, it was never meant to be public. I only see this being used internally.